src/UI/templates/default/Deck/deck.less

```diff
@@ -5,5 +5,19 @@
   > div{
     padding-left: @padding-xs-horizontal;
     padding-right: @padding-xs-horizontal;
+    margin-bottom: @padding-large-vertical;
+  }
+}
+@media (min-width: @screen-md-min) {
+  .row-eq-height {
+
+    display: -webkit-box;
+    display: -webkit-flex;
+    display: -ms-flexbox;
+    display: flex;
+
+    > div {
+      display: flex;
+    }
   }
 }
```

Comment:
a row-eq-height class name sounds like a general approach to make the bootstrap col divs in a bootstrap row of equal height (which would be good to have, if possible). But this would move the responsibility to define the class in the general delos.less (or some other more central place). If it is responsibility of deck.less to define this, the name should be il-deck-row-eq-height or something imo.